packages/perseus-core/src/widgets/interactive-graph/index.ts

```diff
@@ -44,8 +47,28 @@ const interactiveGraphWidgetLogic: WidgetLogic = {
     name: "interactive-graph",
     defaultWidgetOptions,
     getPublicWidgetOptions: getInteractiveGraphPublicWidgetOptions,
-    // TODO(LEMS-3102): add partially inaccessible function
-    accessible: true,
+    // Function determining if a interactive graph is accessible.
+    // Interactive Graphs are accessible as long as:
+    // 1. They do not contain a protractor
+    // 2. They do not contain a graphie background image
+    accessible: (widgetOptions: PerseusWidgetOptions): boolean => {
+        const interactiveGraphOptions =
+            widgetOptions as PerseusInteractiveGraphWidgetOptions;
+
+        // Return false (inaccessible) if the interactive graph contains
+        // a protractor.
+        if (interactiveGraphOptions.showProtractor) {
+            return false;
+        }
+
+        // Return false (inaccessible) if the interactive graph contains
+        // a graphie background image.
+        if (interactiveGraphOptions.backgroundImage?.url?.includes("graphie")) {
+            return false;
+        }
```

Comment:
We have a utility function to detect Graphie URLs: [`isLabeledSVG()`](https://github.com/Khan/perseus/blob/3389e06111b2c801fba6c34ed4e5bdc747ef1ed1/packages/perseus/src/util.graphie.ts#L19) (Perseus thinks of Graphie's as "labelled SVGs").  As with many things in this work we're doing, we'll need to move this function over to `perseus-core`, but it's small and self-contained so should be pretty easy to migrate.